plugins/modules/podman_container_copy.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/python
+# Copyright (c) 2024 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: podman_container_copy
+author:
+  - Alessandro Rossi (@kubealex)
+short_description: Copy file into running container
```

Comment:
```suggestion
short_description: Copy file into running container or from it
```

Reply:
I changed it to "Copy file to/from a container" because it doesn't require to be running.